docs/_includes/frame.html

```diff
@@ -1,7 +1,7 @@
 <table>
-<tr><td style='text-align: center; border: none'>
+<tr><td style='text-align: center; border: none; padding: 0'>
 <iframe src='{{ include.url }}'
-	width='{% if include.width %}{{ include.width }}{% else %}616{% endif %}'
+	width='{% if include.width %}{{ include.width }}{% else %}100%{% endif %}'
```

Comment:
The [HTML validator](https://validator.w3.org/nu/#textarea) returns:

> **Error:** Bad value `100%` for attribute `width` on element `iframe`: Expected a digit but saw `%` instead

I know that this works in Chrome, but does it work in all browsers?